packages/admin-app/src/routes/RouteIds.ts

```diff
@@ -0,0 +1,5 @@
+enum RouteId {
+    Home = '/',
```

Comment:
I know an `Enum` is the first reaction here. Consider a `const` instead. Much more flexible and Enum is getting changed into this object when compiled regardless. 

There is a lot of value for that. Particularly the likelihood of this becoming a function. 

For example:
```js
const routes = {
      home = '/'
      elections = '/elections'
      election_by_id = (id: string) => `/elections/${id}`
}

```

Reply:
Good call, that makes a lot of sense, thanks